### ISSUE TITLE
treeview examples exclude aria-selected, which is required by ARIA 1.1

### ISSUE THREAD

**Issue by stevenyxu** [w3c/aria-practices#1680]
The ARIA 1.1 spec for treeitem requires `aria-selected` https://www.w3.org/TR/wai-aria-1.1/#treeitem.

`aria-selected` is absent on the following authoring practices examples:

https://www.w3.org/TR/wai-aria-practices/examples/treeview/treeview-1/treeview-1a.html
https://www.w3.org/TR/wai-aria-practices/examples/treeview/treeview-2/treeview-2a.html
https://www.w3.org/TR/wai-aria-practices/examples/treeview/treeview-2/treeview-2b.html

This is impacting us because we are attempting to use the ARIA 1.1 tree/treeitem pattern per the spec and are running into AT support issues (e.g., JAWS not announcing selected state, VoiceOver wrongly announcing as a listbox), but lacking an APG reference, it's harder to report bugs.
